hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/recon/ReconConfigKeys.java

```diff
@@ -32,9 +32,11 @@ private ReconConfigKeys() {
 
   public static final String OZONE_RECON_DATANODE_ADDRESS_KEY =
       "ozone.recon.datanode.address";
+  public static final String OZONE_RECON_ADDRESS_KEY =
+      "ozone.recon.address";
```

Comment:
Suggest rename: ozone.recon.server.address

Reply:
I went with ozone.recon.address to keep it consistent with ozone.om.address and ozone.scm.names.